packages/eslint-config-finn-prettier/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "eslint-config-finn-prettier",
+  "version": "1.0.2",
+  "description": "ESLint prettier config for FINN.no",
+  "files": [
+    "index.js",
+    "prettier.js",
+    "react.js"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
+  },
+  "keywords": [
+    "coding-standard",
+    "standards",
+    "eslint",
+    "eslintconfig",
+    "lint",
+    "style",
+    "prettier"
+  ],
+  "repository": "finn-no/eslint-config-finn",
+  "bugs": {
+    "url": "https://github.com/finn-no/eslint-config-finn/issues"
+  },
+  "peerDependencies": {
+    "eslint": "^3.12.2"
+  },
+  "devDependencies": {
+    "eslint": "^3.12.2",
```

Comment:
when you have it as a peer, it's recommended to have it as a dev dep to avoid missing the dependency when developing locally